src/client/camera.cpp

```diff
@@ -454,7 +454,19 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 	if (player->getPlayerControl().zoom && player->getZoomFOV() > 0.001f) {
 		fov_degrees = player->getZoomFOV();
 	} else {
-		fov_degrees = m_cache_fov;
+		// Apply server-sent FOV. If server doesn't
+		// enforce client's FOV, default to m_cache_fov
+		auto fov_spec = player->getFov();
+		if (fov_spec.first > 0.0f) {
+			// If server-sent FOV is a multiplier, multiply
+			// it with m_cache_fov instead of overriding
+			if (fov_spec.second)
+				fov_degrees = 2 * atan(tan(m_cache_fov / 2) * fov_spec.first);
```

Comment:
fov_degrees = m_cache_fov * fov_spec.first;
(i think)

Reply:
In MT 'FOV' is 'view frustrum angle in degrees'.
p_gimeno's transformation is based on considering FOV as something else that is not 'view frustrum angle in degrees'.